utils/objectUtils.js

```diff
@@ -0,0 +1,8 @@
+const has = (obj, path) => {
+  // Regex explained: https://regexr.com/58j0k
+  const pathArray = Array.isArray(path) ? path : path.match(/([^[.\]])+/g)
+
+  return !!pathArray.reduce((prevObj, key) => prevObj && prevObj[key], obj)
```

Comment:
Will do. Thanks!.

Yea, the idea about this reduce is that it's going checking the properties of the object until get the one that the user asked. If at the end we have a object or string or number or array, any of them, then means that the property that the user is asking already exists. If at the end we got undefined means that the property doesn't exists, this is why it is converted to boolean

Here is an example:

First iteration:
```js
{
    test: {
        greetings: 'Hola!'
    }
}
```

Second iteration:
```js
    test: {
        greetings: 'Hola!'
    }
```

Third iteration:
```js
'Hola!'
```

So returns that the object currently exists (!!'Hola!' === true)

Reply:
What about there is an empty string? Maybe it can happen and be a bug in the future

![image](https://github.com/victor-heliomar/i18n-translation-generator/assets/36519478/c912cc05-250c-4322-be32-34da8ee7d383)

Reply:
Hmm makes sense. I think that also could happens if the value of the property is "0". Will add some test cases for it and work on fix it. Good catch!

Reply:
Fixed on this commit! https://github.com/victor-heliomar/i18n-translation-generator/pull/18/commits/75e3fbbc5cd35e99620ed4c65121b28ba593bdea

Reply:
Added more explanatory name here https://github.com/victor-heliomar/i18n-translation-generator/pull/18/commits/6d0a760168cb1164753dd774055126754a5c1c41 ! thanks